.github/workflows/build.yml

```diff
@@ -133,13 +161,19 @@ jobs:
     strategy:
       matrix:
         include:
-          - asset_name: linux
+          - asset_name: linux-amd64
+            artifact_name: capa
+          - asset_name: linux-arm64
             artifact_name: capa
           - asset_name: linux-py312
```

Comment:
```suggestion
          - asset_name: linux-amd64-py312
```

Reply:
No longer relevant due to https://github.com/mandiant/capa/pull/2691#discussion_r2138740671